charts/kong/values.yaml

```diff
@@ -130,6 +130,20 @@ proxy:
     parameters:
     - http2
 
+  # Define stream (TCP) listen
+  # To enable, remove "{}", uncomment the section below, and select your desired
+  # ports and parameters.
+  stream: {}
+    # - name: example-stream
```

Comment:
It makes the example look awkward since everything else has explanatory comments before :)

In practice maybe, as it's used in the SRV record Kubernetes constructs. Probably not too crucial though. It's easy enough to make it default to an auto-generated name with an optional user-specified name if needed later, so defaulting to autogenerated for now.